test/test_torch.py

```diff
@@ -5260,6 +5260,22 @@ def test_multiplication_numpy_scalar(self):
                 self.assertTrue(r2.dtype == t_dtype)
                 self.assertTrue(r2.requires_grad)
 
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    def test_parse_numpy_int(self):
+        # https://github.com/pytorch/pytorch/issues/29252
+        for nptype in [np.int16, np.int8, np.uint8, np.int32, np.int64]:
+            np_arr = np.array([5], dtype=nptype)
+            np_val = np_arr[0]
```

Comment:
nit: all of these can be written as `np_val = np.int64(5)`

Reply:
I want the np_arr below to call torch.from_numpy